app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -39,7 +40,7 @@ const StatusContent = React.createClass({
       } else if (link.textContent[0] === '#' || (link.previousSibling && link.previousSibling.textContent && link.previousSibling.textContent[link.previousSibling.textContent.length - 1] === '#')) {
         link.addEventListener('click', this.onHashtagClick.bind(this, link.text), false);
       } else if (media) {
-        link.innerHTML = '<i class="fa fa-fw fa-photo"></i>';
+        link.innerHTML = '<Icon icon="photo" />';
```

Comment:
sorry @Gargron I stand corrected. I thought that was being parsed by React and assumed you didn't see the new React component. I'll work on fixing that.